### PR TITLE
add new config + bazelrc file to test incompatible flags

### PR DESCRIPTION
**Pull request by nlopezgi**

### CODE REVIEW

File: bazelrc/.bazelrc.test.0.25.0 [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+build:remote --define=EXECUTOR=remote
+build:remote --remote_cache=remotebuildexecution.googleapis.com
+build:remote --remote_executor=remotebuildexecution.googleapis.com
+build:remote --tls_enabled=true
+build:remote --remote_timeout=3600
+build:remote --auth_enabled=true
+
+# Turn on incompatible_enable_cc_toolchain_resolution and don't
+# use crosstool_top
+build:remote --incompatible_enable_cc_toolchain_resolution
```
Comment (xingao267) [inline bazelrc/.bazelrc.test.0.25.0]: Why not use the existing bazelrc and use this flag in the test?

Comment (nlopezgi) [reply bazelrc/.bazelrc.test.0.25.0]: The default bazelrc flag has action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1 which, as I found out doing this test, is incompatible with use of incompatible_enable_cc_toolchain_resolution.

Also, I want to define a way that we can do tests for different versions of Bazel with different incompatible  flags, changing only the minimum set of artifacts for a new Bazel release (i.e., not just for this specific incompatible_enable_cc_toolchain_resolution flag). This way all you will need to add for a new version of bazel (and new set of incompatible flags) is the bazelrc file and the new trigger.

Comment (nlopezgi) [reply bazelrc/.bazelrc.test.0.25.0]: Moved all files to yaml file and removed custom bazelrc file

**Comment by nlopezgi** [comment]
Pending trigger succeeded but did not report back success. This is ready now for review again.
